collector/wifi_linux.go

```diff
@@ -74,6 +76,13 @@ func NewWifiCollector() (Collector, error) {
 			nil,
 		),
 
+		StationInfo: prometheus.NewDesc(
```

Comment:
I just realized that it doesn't matter if the fields are exported or not, because the `wifiCollector` type is unexported anyway. Happy to go either way, though there will be no difference in the public API.